index.js

```diff
@@ -1389,7 +1389,8 @@ module.exports = function({parent}) {
             .then(() => conCreate.close())
             .then(() => this.connection.connect())
             .catch((err) => {
-                this.log && this.log.error && this.log.error({sourcePort: this.config.id, err});
+                err.sourcePort = this.config.id;
```

Comment:
Are you sure that we need this sourcePort at all? Logger already knows the port and if we need to send it in some extra field, then best improve the logger.

Reply:
I don't think we need it. I just left it because it was there beforehand. Now removed. Now removed with this (commit)[https://github.com/softwaregroup-bg/ut-port-sql/pull/50/commits/a971a20a0003d80a398ee41d4bb9b378cf142c3f]